### PR TITLE
Remark about F# not supported

### PR DESCRIPTION
**Pull request by Thraka**
Issue was raised in the dotnet docs about this API. I've filed a fix for the API docs dotnet/roslyn-api-docs#64 and I wanted to sync here.

### CODE REVIEW

**Comment by 333fred** [comment]
@dotnet/roslyn-compiler for a second review.

**Comment by CyrusNajmabadi** [comment]
Sorry, what is meant by "This value is not currently supported by Roslyn."?  Roslyn def supports F#.

**Comment by 333fred** [comment]
@CyrusNajmabadi what do you mean that roslyn definitely supports F#? It definitely doesn't.

**Comment by 333fred** [comment]
It does use some of our language server infrastructure via IVT, but we can neither compile nor emit code, nor provide refactorings, semantic info, or anything of the kind.

**Comment by jmarolf** [comment]
You can write an analyzer in F#, that is supported.  You cannot analyze F# constructs.  I think most people assume that if F# is available in the drop down then they can get an F# syntax tree, which is not possible.

**Comment by CyrusNajmabadi** [comment]
> @CyrusNajmabadi what do you mean that roslyn definitely supports F#? It definitely doesn't.

It definitely does.  When you use F# you're using the Roslyn Workspace system.  You get Roslyn Solution/Project/Documents, etc.

It may not support the full breadth of roslyn features and functionality.  But that's a different statement from "not supported".

**Comment by Thraka** [comment]
Can someone submit an edit to my remarks in the **Files changed** to something clearer? Someone filed an issue on our side asking when a tutorial would be written about F# and roslyn code gen after they noticed this enum value. They were super excited "that F# was supported" and we wanted to clarify in the API docs that this wasn't supported. Or really, clarify what **is** supported.

File: src/Compilers/Core/Portable/Symbols/LanguageNames.cs [hunk 1]
```diff
@@ -20,6 +20,9 @@ public static class LanguageNames
         /// <summary>
         /// The common name used for the F# language.
         /// </summary>
+        /// <remarks>
+        /// This value is not currently supported by Roslyn.
```
Comment (333fred) [inline src/Compilers/Core/Portable/Symbols/LanguageNames.cs]:
```suggestion
        /// F# is not a supported compile target for the Roslyn compiler.
```

@CyrusNajmabadi how about that?

Comment (CyrusNajmabadi) [reply src/Compilers/Core/Portable/Symbols/LanguageNames.cs]: Sounds good :)

**Pull request merged**